app-editors/gvim/gvim-9.0.1341.ebuild

```diff
@@ -0,0 +1,369 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-editors/vim-core and app-editors/vim
+
+VIM_VERSION="9.0"
+VIM_PATCHES_VERSION="9.0.1000"
+
+LUA_COMPAT=( lua5-{1..4} luajit )
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="threads(+)"
+USE_RUBY="ruby27 ruby30 ruby31"
+
+inherit bash-completion-r1 flag-o-matic lua-single prefix python-single-r1 ruby-single toolchain-funcs vim-doc xdg-utils
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vim/vim.git"
+	EGIT_CHECKOUT_DIR=${WORKDIR}/vim-${PV}
+else
+	SRC_URI="https://github.com/vim/vim/archive/v${PV}.tar.gz -> vim-${PV}.tar.gz
+		https://gitweb.gentoo.org/proj/vim-patches.git/snapshot/vim-patches-vim-${VIM_PATCHES_VERSION}-patches.tar.bz2"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
+fi
+S="${WORKDIR}"/vim-${PV}
+
+DESCRIPTION="GUI version of the Vim text editor"
+HOMEPAGE="https://www.vim.org https://github.com/vim/vim"
+
+LICENSE="vim"
+SLOT="0"
+IUSE="acl aqua crypt cscope debug lua minimal motif netbeans nls perl python racket ruby selinux session sound tcl"
+REQUIRED_USE="
+	lua? ( ${LUA_REQUIRED_USE} )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	aqua? ( !motif )
+"
+
+RDEPEND="
+	~app-editors/vim-core-${PV}
+	>=app-eselect/eselect-vi-1.1
+	>=sys-libs/ncurses-5.2-r2:0=
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXext
+	x11-libs/libXt
+	acl? ( kernel_linux? ( sys-apps/acl ) )
+	!aqua? (
+		motif? ( >=x11-libs/motif-2.3:0 )
+		!motif? (
+			x11-libs/gtk+:3
+			x11-libs/libXft
+		)
+	)
+	crypt? ( dev-libs/libsodium:= )
+	cscope? ( dev-util/cscope )
+	lua? (
+		${LUA_DEPS}
+		$(lua_gen_impl_dep 'deprecated' lua5-1)
+	)
+	nls? ( virtual/libintl )
+	perl? ( dev-lang/perl:= )
+	python? ( ${PYTHON_DEPS} )
+	racket? ( dev-scheme/racket )
+	ruby? ( ${RUBY_DEPS} )
+	selinux? ( sys-libs/libselinux )
+	session? ( x11-libs/libSM )
+	sound? ( media-libs/libcanberra )
+	tcl? ( dev-lang/tcl:0= )
+"
+DEPEND="${RDEPEND}"
+# configure runs the Lua interpreter
```

Comment:
Needs to DEPEND on xorg-proto too.

Reply:
Coolio, I'll take a look (replying to all of the above).